Core/clim-core/standard-presentations.lisp

```diff
@@ -129,28 +129,29 @@
 (define-presentation-type blank-area ()
   :inherit-from t)
 
+;;; Do other slots of this have to be bound in order for this to be
+;;; useful?  Guess we'll see.
+(defconstant *null-presentation* (make-instance 'standard-presentation
+                                                :object nil
+                                                :type 'blank-area
+                                                :view +textual-view+))
```

Comment:
Why the change to `defconstant`? If we ever remove the custom `defconstant` and switch to `cl:defconstant` (I think there was a plan to do so at some point), this will lead to problems.

Reply:
this variable is specified to be a constant

Reply:
I think deviating from the spec is best in this case since our `defconstant` does not make `*null-presentation*` a constant, but using `cl:defconstant` will lead to problems (requiring `make-load-form` methods, object identify issues).

It is strange that the name uses earmuffs while for example `+black+` uses `+`.

Reply:
I'll change it to defvar then, to avoid at least reinitialization during recompile.

Reply:
changed.